bin/nic.pl

```diff
@@ -313,16 +313,23 @@ sub getHomeDir {
 }
 
 sub loadConfig {
-	open(my $cfh, "<", getHomeDir()."/.nicrc") or return;
-	while(<$cfh>) {
-		if(/^(.+?)\s*=\s*\"(.*)\"$/) {
-			my $key = $1;
-			my $value = $2;
-			$CONFIG{$key} = $value;
-		}
-	}
+    open(my $cfh, "<", getHomeDir()."/.nicrc") or return;
+    while(<$cfh>) {
+        # Remove leading and trailing whitespace
+        s/^\s+|\s+$//g;
+        if (/^(.+?)\s*=\s*"(.*)"$/) {
+            my $key = $1;
+            my $value = $2;
+            # Remove leading and trailing whitespace from key and value
+            $key =~ s/^\s+|\s+$//g;
+            $value =~ s/^\s+|\s+$//g;
```

Comment:
This seems like a worthwhile change, but I'm not sure if format-correcting the user configuration at load is something we want to be responsible for or not. Will leave it up to an additional set of eyes, but `dm.pl` should catch anything that's wrong with the control config and the other options are boolean and appear to work as expected irrespective of whitespace.

Reply:
Re other options: https://theos.dev/docs/nic#nicrc